src/language-markdown/print-whitespace.js

```diff
@@ -106,109 +29,35 @@ function isInSentenceWithCJSpaces({ parent: sentenceNode }) {
  *
  *     "You might want to use Prettier."
  *
- * However, Chinese and Japanese don't use U+0020 Space to divide words, so line
- * breaks shouldn't be replaced with spaces for those languages.
+ * HTML specify that `\n` between letters must be converted to a space, so
+ * this function always returns `true` until the HTML specification is
+ * revised.
  *
- * PRs are welcome to support line breaking rules for other languages.
- *
- * @param {AstPath} path
- * @param {boolean} isLink
  * @returns {boolean}
  */
-function lineBreakCanBeConvertedToSpace(path, isLink) {
-  if (isLink) {
-    return true;
-  }
+function lineBreakCanBeConvertedToSpace() {
```

Comment:
Do you think we should remove this function?

Reply:
It depends on whether we'll have to add it again.
I can't predict when all browsers will change their behavior.